community/cypher/cypher/src/test/scala/org/neo4j/cypher/LoadCsvCompressionAcceptanceTest.scala

```diff
@@ -73,19 +73,25 @@ class LoadCsvCompressionAcceptanceTest extends ExecutionEngineFunSuite with Befo
     * Simple server that handles csv requests in plain text, gzip and deflate
     */
   private class TestServer {
-    private val server: Server = new Server(8080)
-    private val handlers = new ContextHandlerCollection()
-    addHandler("/csv", new CsvHandler)
-    addHandler("/gzip", new GzipCsvHandler)
-    addHandler("/deflate", new DeflateCsvHandler)
-    server.setHandler(handlers)
+
+     //let jetty pick a random available port for us
+     private var _port = 0
+     private val server: Server = new Server(_port)
```

Comment:
If we always want to initialise the Server with port 0, can't we not use the _port variable here? I find it slightly confusing